string first_name John
string last_name Doe
